### PR TITLE
Add possibility to configure single sign on through puppet

### PR DESCRIPTION
**Pull request by jurgenmehja**
This PR adds the required parameters to implements single sign-on

### CODE REVIEW

File: manifests/dashboard.pp [hunk 1]
```diff
@@ -14,6 +14,16 @@
   $dashboard_server_host = '0.0.0.0',
   $dashboard_server_hosts = "https://${indexer_server_ip}:${indexer_server_port}",
 
+  # Parameters used for sso login
+  $enable_sso_login = undef,
```
Comment (poloz-lab) [inline manifests/dashboard.pp:18]: The variable is named "enable_sso_login" but the code manage only openid.
With this parameter name, I expect to be able to use also SAML.

File: templates/wazuh_dashboard_yml.erb [hunk 2]
```diff
@@ -9,9 +9,16 @@ opensearch.password: <%= @dashboard_password %>
...
+opensearch_security.openid.client_secret: <%= @opensearch_security_openid_client_secret %>
+opensearch_security.openid.base_redirect_url: <%= @opensearch_security_openid_base_redirect_url %>
+opensearch_security.openid.verify_hostnames: <%= @opensearch_security_openid_verify_hostnames %>
+<% end -%>
 server.ssl.enabled: true
 server.ssl.key: "<%= @dashboard_path_certs %>/dashboard-key.pem"
 server.ssl.certificate: "<%= @dashboard_path_certs %>/dashboard.pem"
 opensearch.ssl.certificateAuthorities: ["<%= @dashboard_path_certs %>/root-ca.pem"]
 uiSettings.overrides.defaultRoute: /app/wazuh
-
```
Comment (poloz-lab) [inline templates/wazuh_dashboard_yml.erb:17]: Trailing newline removed

**Review by poloz-lab — COMMENTED** [review]
Hi @jurgenmehja ,

I'm not a maintainer but I still want to share my review.
Take it or leave it ;)

**Review by vcerenu — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]

**Pull request merged**